packages/client/src/hooks/useSetIsRead.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import { MutationResult } from "react-apollo"
+import * as graphql from "../generated/graphql"
+
+export default function useSetisRead(
```

Comment:
This is a custom hook; this pattern is highly useful for encapsulating logic to keep components as simple as possible. A custom hook is a function. The name always starts with `use` so that ESLint can correctly make check uses of the hook.

Notice that a custom hook can call other hooks. Hooks only work when they are called either from function components or from custom hooks.